tables/automl/notebooks/energy_price_forecasting/energy_price_forecasting.ipynb

```diff
@@ -0,0 +1,1204 @@
+              "ok": true,
+              "headers": [
+                [
+                  "content-type",
+                  "application/javascript"
+                ]
+              ],
+              "status": 200,
+              "status_text": ""
+            }
+          },
+          "base_uri": "https://localhost:8080/",
+          "height": 602
+        },
+        "outputId": "4d3628f9-e5be-4145-f550-8eaffca97d37"
+      },
+      "cell_type": "code",
+      "source": [
+        "#@title Install AutoML Tables client library { vertical-output: true }\n",
+        "\n",
+        "from __future__ import absolute_import\n",
+        "from __future__ import division\n",
+        "from __future__ import print_function\n",
+        "\n",
+        "from google.colab import files\n",
+        "import tarfile\n",
+        "\n",
+        "# Upload the client library\n",
+        "compressed_file_upload = files.upload()\n",
+        "compressed_file_name = list(compressed_file_upload.keys())[0]\n",
+        "# Decompress the client library\n",
+        "with tarfile.open(compressed_file_name) as tar:\n",
+        "  tar.extractall(path='.')\n",
+        "# Install the client library\n",
```

Comment:
Remove the lines "\n, from _future_ importabsolute_import\n" through #install the client library\n" included.
we need to use the public packager